shell.c

```diff
@@ -306,16 +309,94 @@ char* readLine() {
     }
 }
 
+// Display result of the assignment
+struct pre_assessment {
+    bool valid;
+    char *assignment;
+};
+
+void display_assignment_result(struct pre_assessment pa, char *line) {
```

Comment:
Let's use camel case for consistency.

Reply:
I think you have misunderstood the naming convention for C. Camel case is used in Java but not in C. You can use this link for the reference (https://users.ece.cmu.edu/~eno/coding/CCodingStandard.html#stacknames).

Reply:
It is a standart not a rule. What I want to say that if you want to use the snake case you should change all the places to keep the consistency.

Reply:
Yes, I know that it is a standard but it is better that we use the latter because our code is submitted and will be flagged when marking on this issue. We need to stay close to the standard to avoid any risen issues and show that we understand C better. In addition, I don’t change your parts because there would be too much and it might cause errors and it would interfere with mine and I would have a hard time figuring out the issue. In fact, there was a tiny bit where you forgot to put a function on the top and the compilation stage prompted a warning so I just fixed that bit to ensure when I compiled there was no more error.

Reply:
To my mind using snake case do not solve any logical issues or show your knowledge in C:) It is strange that you have got a warning though. Maybe it is because of different compilers/compiler versions.